### ISSUE TITLE
[Accessibility] Audio Cues still doesn't work in github.dev insider

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#165357]
CC @isidorn

Following #147190

I think the latest GitHub Dev insiders has already picked up the audio cues web patch, given that I can see the latest audio cues in the list, such as "Task Completed" and "Task Failed"

![image](https://user-images.githubusercontent.com/19754711/199735912-74caa892-b634-4553-80ef-a2fb6c190d61.png)

## sysem Info

Version: 1.74.0-insider
Commit: 86f0f9bfa268fe21af7efae37d8f0da1dba8137f
Date: 2022-11-03T05:23:43.231Z
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36
## Reproducible Steps

1. Go to https://github.dev/microsoft/vscode

1. Launch the GitHub Dev with insider mode.

1. Launch "Help: List Audio Cues"

* Notes: audio cues are not playing.

**Comment by isidorn** [comment]
@meganrogge I suggest to look into Developer Tools. And to double check if mp3 files are part of our vscode.dev bundle.

**Comment by meganrogge** [comment]
This works for me via Safari on mac. I'll see I can reproduce the issue on windows

**Comment by jooyoungseo** [comment]
FYI, I tested with Chrome in Windows, and it did not work.

**Comment by meganrogge** [comment]
Works on windows when using edge but not chrome

**Comment by meganrogge** [comment]
![error](https://user-images.githubusercontent.com/29464607/199806080-19f5ce9c-6656-415f-861b-0fa52cdfe7b1.PNG)

**Comment by meganrogge** [comment]
Turning off `Windows Firewall` fixes it

**Comment by jooyoungseo** [comment]
@meganrogge  -- Would there be any way to exclude github.dev from the firewall rule? I need to keep it on for security purpose.

**Comment by meganrogge** [comment]
@jooyoungseo yes you can do that in the `Windows Firewall with advanced security` section by creating a rule

**Comment by meganrogge** [comment]
an alternative would be to use Edge instead of Chrome

**Comment by jooyoungseo** [comment]
No success with Firewall off:

![image](https://user-images.githubusercontent.com/19754711/199810094-b085a839-6539-4d2a-8479-909a5f56c96b.png)

**Comment by jooyoungseo** [comment]
Also Edge does not produce sound for Audio Cues on my end. Odd...

**Comment by meganrogge** [comment]
Are there errors in the dev tools console?

**Comment by jooyoungseo** [comment]
Sorry. I will  keep you posted later on that. BTW, [vscode.dev](https://vscode.dev) doesn't even require the Firewall trick for the audio cues in chrome. It's playing from without any settings. What's the difference? Would there be any way to make GitHub Dev  works just like vscode.dev out of the box?

**Comment by meganrogge** [comment]
The bundle must be different for vscode.dev & github.dev because the former works in most cases while the latter fails.

![Screen Shot 2022-11-03 at 12 48 50 PM](https://user-images.githubusercontent.com/29464607/199820067-a31872f4-a682-4039-bfa8-e2093775f344.png)

**Comment by meganrogge** [comment]
more interesting part of the error:

```
meganrogge-microsoft-vscode-8gj2.github.dev/:1 
        
       Refused to load media from 'https://main.vscode-cdn.net/insider/8fa188b2b301d36553cbc9ce1b0a146ccb93351f/out/vs/workbench/contrib/audioCues/browser/media/error.mp3' because it violates the following Content Security Policy directive: "media-src 'none'".
```

**Comment by meganrogge** [comment]
Codespaces sets a content policy of none and will need to change that for this to work

**Comment by meganrogge** [comment]
Got confirmation from codespaces that this will be fixed soon, closing as this is entirely upstream and we cannot do anything about it

**Comment by meganrogge** [comment]
Hopefully this can be verified as fixed in November

**Comment by jooyoungseo** [comment]
Thanks, @meganrogge ! I will keep checking this feature.going forward.

**Comment by joyceerhl** [comment]
This cannot be verified as the upstream issue is not fixed, I have optimistically moved to December 2022 so we verify it then.

**Comment by joyceerhl** [comment]
This issue still cannot be verified as the upstream issue has not been fixed

**Comment by joaomoreno** [comment]
There were two issues here. One was the CSP which was already addressed by Codepsaces.

The reason why this hasn't been able to be verified so far is because the other issue was never addressed.

Fixed by https://github.com/microsoft/vscode/pull/174577, more info in https://github.com/microsoft/vscode-dev/issues/826
